### PR TITLE
Building Clouds

### PR DESCRIPTION
**Pull request by MassterKek**
Still having an issue getting the circles to appear, can't figure out why they won't show.

### CODE REVIEW

File: js/site.js [hunk 1]
```diff
@@ -49,6 +48,56 @@ function draw() {
...
+}
+
+function generatePositionValues(pre_pos) {
+  console.log("generatePositionValues() called successfully");
+  const rad = 50 - 25*Math.random();
+  const vector_min = Math.max(rad, pre_pos.rad);
+  const vector_max = rad + pre_pos.rad;
+  const vector_size = remapNumber(Math.random(), 0, 1, vector_min, vector_max);
+  const vector_angle = remapNumber(Math.random(), 0, 1, 225, 315);
+  const xpos = Math.cos(vector_angle)*vector_size;
```
Comment (cronin4392) [inline js/site.js]: I think you have x and y flipped. I think x should be sin and y cos?

**Comment by cronin4392** [comment]
Looking good, approved the PR. @MassterKek you can merge whenever you're ready. I'd recommend doing "Squash and merge" from the merge button dropdown.

**Review by cronin4392 — APPROVED** [review]

File: js/site.js [hunk 2]
```diff
@@ -78,31 +79,33 @@ function cloudSVG(n, precip) {
...
+  console.log(`<circle cx="${xpos}px" cy="${ypos}px" r="${rad}px" fill="${p}"></circle>`);
   return `<circle cx="${xpos}px" cy="${ypos}px" r="${rad}px" fill="${p}"></circle>`;
 }
 
+// Sets hsl color for cloud
 function cloudColor(n) {
   let c = remapNumber(n, 0, 200, 25, 85);
   let col_val = 110 - c;
-  return `hsl(0%, 0%, ${col_val}%)`;
+  console.log(`hsl(0, 0%, ${col_val}%)`);
```
Comment (cronin4392) [inline js/site.js]: Out of curiosity @MassterKek, are you using these console logs for debugging? If so, I'd keep them, but if you're just adding out of habit I'd recommend removing them. Just adds noise.

File: js/site.js [hunk 3]
```diff
@@ -67,7 +68,7 @@ function generatePositionValues(pre_pos) {
 }
 
 // Creates an svg containing n circles
-function cloudSVG(n, precip) {
+function cloudSVG(n, p) {
```
Comment (cronin4392) [inline js/site.js]: I personally tend to try and use somewhat descriptive variable names. Helps with debugging and comprehending the code later.

**Review by cronin4392 — APPROVED** [review]

**Comment by cronin4392** [comment]
When you merge it, just remember to take out the [WIP] from the title!

File: js/site.js (hunk 2 above)
Comment (MassterKek) [reply js/site.js]: They were for debugging, was going to clear them later

File: js/site.js (hunk 3 above)
Comment (MassterKek) [reply js/site.js]: Noted, will adjust

File: css/styles.css [hunk 4]
```diff
@@ -1,20 +1,19 @@
...
   }
 
   50% {
-    transform: translate3d(5%, 0, 0);
+    transform: translate3d(0, 1%, 0);
   }
 }
 
 circle {
+  position: absolute;
```
Comment (cronin4392) [inline css/styles.css]: remove?

File: js/site.js [hunk 5]
```diff
@@ -17,14 +19,16 @@ fetch(api)
...
+      precip_mm,
       temp_f,
       feelslike_f,
       humidity,
       vis_miles,
       wind_mph,
       temp_c,
     } = data.current;
-
+    precip = data.current.precip_mm;
```
Comment (cronin4392) [inline js/site.js]: You can rename a variable when you pull it out of the object above, for example:

```
const { precip_mm: precip, } = data.current;
```

**Review by cronin4392 — CHANGES_REQUESTED** [review]

**Pull request merged**